packages/templates/app/typescript-react/package.json

```diff
@@ -14,8 +14,8 @@
     "@polywrap/react": "0.10.0-pre.6",
     "http-proxy": "1.18.1",
     "http-proxy-middleware": "1.0.6",
-    "react": "16.13.1",
```

Comment:
Why the downgrade?

Reply:
It was an issue before but I guess it got solved later so can revert that